### ISSUE TITLE
CAD2016 dataset

### ISSUE THREAD

**Issue by Luochangjiang10** [WenjunHui1/TSP-SAM#5]
Hello, this is a great work. I would like to know how you divide the dataset on the CAD2016 dataset. Can you provide it? Thank you very much.

**Comment by WenjunHui1** [comment]
It was shown in the implementation details of the paper.

**Comment by Luochangjiang10** [comment]
On the CAD2016 dataset, we adopt a ratio of 9:1 to divide all classes into training and testing sets and report the mean value over the testing set.Does this mean that if there are 20 images of frogs, 18 images will be selected as training set and 2 images will be selected as test set?Thank you for your reply.

**Comment by Luochangjiang10** [comment]
I feel it is very ambiguous. I hope you can give me an answer.
